Src/libCZI/CziUtils.cpp

```diff
@@ -20,6 +20,8 @@
 // 
 //******************************************************************************
 
+#include <limits>
```

Comment:
This should go after `#include "stdafx.h"`. Precompiled header include needs to go first.

Reply:
Moved